### PR TITLE
glass: Updates language assets

### PR DESCRIPTION
**Pull request by Devp00l**
Signed-off-by: Stephan Müller <smueller@suse.com>
Co-authored-by: Volker Theile <vtheile@suse.com>

## Checklist
- [ ] References issues, create if required
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins test tumbleweed`
- `jenkins run tumbleweed`
- `jenkins test leap`
- `jenkins run leap`

</details>

### CODE REVIEW

File: src/glass/src/app/i18n.helper.ts [hunk 1]
```diff
@@ -35,8 +36,10 @@ export class TranslateHttpLoader implements TranslateLoader {
 
 export const supportedLanguages: Record<string, string> = {
   /* eslint-disable @typescript-eslint/naming-convention */
-  en_US: 'English',
-  de_DE: 'Deutsch'
+  en_US: TEXT('English'),
```
Comment (votdev) [inline src/glass/src/app/i18n.helper.ts:39]: I'm not sure if we should translate them. Imagine you accidentally switch to Chinese language  and then `Deutsch` is displayed in Chinese letters. Each language should be hard-coded:
```
en_US => English
de_DE => Deutsch
zh_CN (Chinese) => 中國人
zh_TW (Taiwanese) => 台灣
```
@AvengerMoJo Please correct my Google translations.

**Review by votdev — CHANGES_REQUESTED** [review]

File: src/glass/src/app/i18n.helper.ts (hunk 1 above)
Comment (votdev) [reply src/glass/src/app/i18n.helper.ts:39]: According to @AvengerMoJo you should use
```
zh_CN (Simplified) => 简体中文
zh_TW (Traditional) => 繁體中文
```

**Review by AvengerMoJo — COMMENTED** [review]
It does not allow me to comment before I review I believe.
```
zh_CN (Simplified) => 简体中文
zh_TW (Traditional) => 繁體中文
```
Let's use the above translation

**Comment by AvengerMoJo** [comment]
jenkins test tumbleweed

**Review by AvengerMoJo — APPROVED** [review]

**Review by votdev — DISMISSED** [review]

File: src/glass/src/assets/i18n/de_DE.json [hunk 2]
```diff
@@ -1,73 +1,144 @@
...
+   "15 min": "15 min",
+   "5 min": "5 min",
+   "Add": "Hinzufügen",
+   "Address": "Adresse",
+   "Allocated Size": "Zugewiesene Größe",
    "Aquarium": "Aquarium",
+   "Aquarium will now install and setup the core system. All recognized storage devices are automatically assigned. Afterwards, you will be able to configure the services you want to provide.": "",
    "Architecture": "Architektur",
-   "Assigned": "Zugewiesen",
+   "Authentication Token": "Authentifizierungstoken",
```
Comment (s0nea) [inline src/glass/src/assets/i18n/de_DE.json:11]:
```suggestion
   "Authentication Token": "Authentifizierungsschlüssel",
```

File: src/glass/src/assets/i18n/de_DE.json [hunk 3]
```diff
@@ -1,73 +1,144 @@
...
    "Available": "Verfügbar",
+   "Available Capacity": "Verfügbare Kapazität",
    "Back": "Zurück",
-   "Bootstrap system": "",
-   "CPU Load": "CPU-Auslastung ",
-   "Cancel": "Beenden",
+   "Bootstrapping not possible at the moment, please retry later.": "",
+   "CPU Load": "CPU Load",
```
Comment (s0nea) [inline src/glass/src/assets/i18n/de_DE.json:16]: Why not keeping "CPU-Auslastung"?

File: src/glass/src/assets/i18n/de_DE.json [hunk 4]
```diff
@@ -1,73 +1,144 @@
...
+   "Cancel": "Abbruch",
    "Capacity": "Kapazität",
    "CephFS": "CephFS",
-   "Choose installation mode": "Wählen Sie den Installationsmodus ",
-   "Choose selected devices": "Wählen Sie ausgewählte Geräte aus ",
-   "Configure CephFS service": "Konfigurieren Sie den CephFS-Dienstes",
-   "Configure NFS service": "Konfigurieren Sie den NFS-Dienst",
-   "Configure iSCSI service": "Konfigurieren Sie den iSCSI-Dienst",
+   "CephFS Service": "CephFS Bedienung",
```
Comment (s0nea) [inline src/glass/src/assets/i18n/de_DE.json:20]:
```suggestion
   "CephFS Service": "CephFS-Dienst",
```

File: src/glass/src/assets/i18n/de_DE.json [hunk 5]
```diff
@@ -1,73 +1,144 @@
...
    "Continue": "Fortsetzen",
+   "Copied text to the clipboard successfully.": "Text erfolgreich in die Zwischenablage kopiert.",
+   "Copy to clipboard": "In die Zwischenablage kopieren",
    "Create": "Erstellen",
    "Create new cluster": "Neuen Cluster erstellen",
-   "Dashboard": "",
-   "Deploy a service": "Einen Dienst bereitstellen",
-   "Failed to bootstrap the system.": "",
+   "Current deployment has a single host and can't tolerate failures.": "",
+   "Dashboard": "Dashboard",
```
Comment (s0nea) [inline src/glass/src/assets/i18n/de_DE.json:31]: Can't we just leave it empty so it falls back to the English translation?

File: src/glass/src/assets/i18n/de_DE.json [hunk 6]
```diff
@@ -1,73 +1,144 @@
...
    "Sorry, we could not find what you are looking for.": "",
-   "Space": "",
    "Start": "Start",
-   "System Information": "",
-   "The step will erase all data on the listed devices. Are you sure you want to continue?": "",
-   "Toggle navigation": "",
-   "Type": "",
-   "Unassigned": "Nicht zugewiesen",
+   "Summary": "",
+   "System Information": "System Information",
```
Comment (s0nea) [inline src/glass/src/assets/i18n/de_DE.json:108]:
```suggestion
   "System Information": "Systeminformationen",
```

File: src/glass/src/assets/i18n/de_DE.json [hunk 7]
```diff
@@ -1,73 +1,144 @@
...
+   "Unable to finish deployment.": "",
    "Uptime": "Betriebszeit",
+   "Usable Size": "",
+   "Use a NTP host on the internet": "",
+   "Use the following command line to mount the file system.": "",
+   "Use this token to authenticate a new node when adding it to the cluster.": "",
+   "Use your own NTP host": "",
    "Used": "Benutzt",
-   "Vendor": "",
+   "Vendor": "Lieferant",
```
Comment (s0nea) [inline src/glass/src/assets/i18n/de_DE.json:134]:
```suggestion
   "Vendor": "Anbieter",
```

**Comment by jecluis** [comment]
@votdev is this being merged in the next few hours? I'd like to cut M4 soon.

**Review by s0nea — APPROVED** [review]

**Review by votdev — APPROVED** [review]

**Review by s0nea — APPROVED** [review]

**Comment by jecluis** [comment]
jenkins run tumbleweed

**Pull request merged**